benchmarks/cugraph/standalone/cugraph_bulk_sampling.py

```diff
@@ -0,0 +1,647 @@
+    print(f"Max Peak to output graph ratio across workers = {output_to_peak_ratio:.2f}")
+    print(
+        f"Max Peak to avg input graph ratio across workers = {input_to_peak_ratio:.2f}"
+    )
+    return (
+        input_to_peak_ratio,
+        output_to_peak_ratio,
+        input_memory_per_worker,
+        peak_allocation_across_workers,
+    )
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    
+    parser.add_argument(
+        '--output_root',
+        type=str,
+        help='The output root directory.  File/folder names are auto-generated.',
+        required=True,
+    )
+
+    parser.add_argument(
+        '--dataset_root',
+        type=str,
+        help='The dataset root directory containing ogb datasets.',
+        required=True,
+    )
+
+    parser.add_argument(
+        '--datasets',
+        type=str,
+        help=(
+            'Comma separated list of datasets; can specify ogb or rmat (i.e. ogb_papers100M[2],rmat_22_16).'
+            ' For ogb datasets, can provide replication factor using brackets.'
+        ),
+        required=True,
+    )
+
+    parser.add_argument(
+        '--fanouts',
+        type=str,
+        help='Comma separated list of fanouts (i.e. 10_25,5_5_5)',
+        required=False,
+        default='10_25',
+    )
+
+    parser.add_argument(
+        '--batch_sizes',
+        type=str,
+        help='Comma separated list of batch sizes (i.e. 500,1000)',
+        required=False,
+        default='500,1000'
+    )
+
+    parser.add_argument(
+        '--seeds_per_call_opts',
+        type=str,
+        help='Comma separated list of seeds per call (i.e. 1000000,2000000)',
+        required=False,
+        default='500000,1000000,2000000',
+    )
+    
+    parser.add_argument(
+        '--reverse_edges',
+        action='store_true',
+        help='Whether to reverse the edges for DGL (defaults to False).  Should be True for DGL, False for PyG.',
+        required=False,
+        default=False,
+    )
+
+    parser.add_argument(
+        '--dask_worker_devices',
+        type=str,
+        help='Comma separated list of dask worker devices',
+        required=False,
+        default=[0,1]
+    )
+
+    parser.add_argument(
+        '--random_seed',
+        type=int,
+        help='Random seed',
+        required=False,
+        default=62
+    )
+
+    return parser.parse_args()
+
+
+# call __main__ function
+if __name__ == "__main__":
+    logging.basicConfig()
+
+    args = get_args()
+    fanouts = [[int(f) for f in fanout.split('_')] for fanout in args.fanouts.split(',')]
+    datasets = args.datasets.split(',')
+    batch_sizes = [int(b) for b in args.batch_sizes.split(',')]
+    seeds_per_call_opts = [int(s) for s in args.seeds_per_call_opts.split(',')]
+    dask_worker_devices = [int(d) for d in args.dask_worker_devices.split(',')]
+
+    client, cluster = start_dask_client(dask_worker_devices=dask_worker_devices, jit_unspill=False, rmm_pool_size=28e9, rmm_async=True)
```

Comment:
Below does not work as `rmm_async` is not available in `start_dask_client`
```suggestion
    client, cluster = start_dask_client(dask_worker_devices=dask_worker_devices, jit_unspill=False, rmm_pool_size=28e9, rmm_async=True)
```

Reply:
oh, that's part of another PR that isn't merged yet

Reply:
it should not be part of thisone